xnmt/experiment.py

```diff
@@ -51,7 +51,7 @@ def __call__(self, save_fct):
     eval_scores = ["Not evaluated"]
     if self.train:
       logger.info("> Training")
-      save_fct() # save initial model
+      # save_fct() # save initial model
```

Comment:
Does this work even when setting ```run_for_epochs``` to 0? If so I think it's ok to out-comment this for now. Supporting ```run_for_epochs=0``` is important for inference-only configs because with multi-tasking one can't set ```train=None``` because the models are sub-objects of the training regimen.

Reply:
maybe we just save the model in the above cases? I find it is also annoying to wait at the beginning of training...

Reply:
Yeah you're right.. actually I think there's no need to save an inference-only model in the first place. I changed the code to skip the ```revert_to_best_model()``` in case it has never been saved.